src/main/java/org/dcsa/ebl/service/impl/ShippingInstructionServiceImpl.java

```diff
@@ -348,7 +348,7 @@ Mono<List<Booking>> validateDocumentStatusOnBooking(ShippingInstructionTO shippi
         .flatMap(
             carrierBookingReference ->
                 bookingRepository
-                    .findAllByCarrierBookingReference(carrierBookingReference)
+                    .findAllByCarrierBookingReferenceWhereValidUntilIsNull(carrierBookingReference)
```

Comment:
I think it is wise to align the errors with the errors described in this story: https://dcsa.atlassian.net/browse/DDT-957

Reply:
Done, created a new PR for event-core changes: https://github.com/dcsaorg/DCSA-Event-Core/pull/316